examples/migrations/langgraph-checkpoints/artifacts/runs/20260722T223523Z-27da3254/memanto-roundtrip-okf/sessions/langgraph-migration-20260722t223523z-27da3254_2026-07-22_unknown_summary.md

```
@@ -0,0 +1,173 @@
+# Session Summary for langgraph-migration-20260722t223523z-27da3254
+**Session ID:** `unknown`
+
+---
+
+### [2026-07-22 22:35:24] [ARTIFACT] LangGraph transcript: product-launch
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)